lib/tasks/plugin_assets.rake

```diff
@@ -128,7 +128,11 @@ task 'plugin:assets:precompile', [:plugin] => [:environment] do |t, args|
       def compile
         return unless File.exist?("#{@plugin.path}/package.json")
         ENV["NODE_ENV"] ||= 'production'
-        webpack_bin = ::Rails.root.join(::Rails.configuration.webpack.binary)
+        if system('which webpack')
```

Comment:
I wonder if this is the best place. Should we check if `::Rails.configuration.webpack.binary` exists and otherwise fall back to `webpack`? Also wonder if it should be done in https://github.com/theforeman/foreman/tree/develop/config/environments so we always use the correct webpack binary rather than just in this rake task.